randovania/games/prime1/gui/ui_files/preset_prime_qol.ui

```diff
@@ -268,6 +268,20 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="QCheckBox" name="remove_bars_great_tree_hall_check">
+             <property name="text">
+              <string>Remove Bars in Great Tree Hall</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="remove_bars_great_tree_hall_label">
+             <property name="text">
+              <string>Removes the boost ball bars obstacle in Tallon Overworld Great Tree Hall</string>
```

Comment:
```suggestion
              <string>Removes the Boost Ball bars obstacle in Tallon Overworld's Great Tree Hall</string>
```

Potentially even with ", allowing free movement between the lower and upper levels of the room"